File: app/src/main/java/org/cru/godtools/adapter/EmptyListHeaderFooterAdapter.java
package org.cru.godtools.adapter;

import androidx.annotation.NonNull;

public class EmptyListHeaderFooterAdapter extends BaseHeaderFooterAdapter {
    public static class Builder extends BaseHeaderFooterAdapter.Builder<Builder> {
        public EmptyListHeaderFooterAdapter build() {
            return new EmptyListHeaderFooterAdapter(this);
        }
    }

    EmptyListHeaderFooterAdapter(@NonNull final Builder builder) {
        super(builder);
    }
}

File: app/src/main/java/org/cru/godtools/adapter/EmptyListHeaderFooterAdapter.kt
package org.cru.godtools.adapter

class EmptyListHeaderFooterAdapter internal constructor(builder: Builder) : BaseHeaderFooterAdapter(builder) {
    class Builder : BaseHeaderFooterAdapter.Builder<Builder>() {
        fun build(): EmptyListHeaderFooterAdapter {
            return EmptyListHeaderFooterAdapter(this)
        }
    }
}
